lib/puppet_x/chocolatey/chocolatey_common.rb

```diff
@@ -62,6 +62,10 @@ def choco_config_file
         chocoInstallPath = PuppetX::Chocolatey::ChocolateyInstall.install_path
         choco_config = "#{chocoInstallPath}\\config\\chocolatey.config"
 
+        # choco may be installed, but a config file doesn't exist until the
+        # first run of choco - trigger that by checking the version
+        choco_run_ensure_config = choco_version
```

Comment:
Would it be better to add an additional file existance check above as well?  That way we don't run a choco.exe IF the config file exists